src/Microsoft.Health.Fhir.Liquid.Converter/Utilities/TemplateUtility.cs

```diff
@@ -145,12 +146,12 @@ public static Template ParseJsonSchemaTemplate(string content)
                 return null;
             }
 
-            JSchema schema;
+            JsonSchema schema;
             try
             {
-                schema = JSchema.Parse(content);
+                schema = JsonSchema.FromJsonAsync(content).GetAwaiter().GetResult();
             }
-            catch (JSchemaReaderException ex)
+            catch (JsonReaderException ex)
```

Comment:
Just curious when will this exception throw? Will the `FromJsonAsync` method throw other type of exceptions?

Reply:
Updated to catch **Exception**